### PR TITLE
rtl_433 0.01 (new formula)

### PR DESCRIPTION
**Pull request by labeneator**
### All Submissions:

- [ ] Have you followed the guidelines in our [Contributing]
(https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
Yes
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?
Yes

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [ ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
Yes
- [ ] Have you built your formula locally prior to submission with `brew install <formula>`?
Yes

### Changes to Homebrew's Core:

RTL 433 is a tool that uses librtlsdr to talk to 433Mhz
wireless devices such as weather stations, wireless alarms,
wireless doorbells...

### CODE REVIEW

File: Library/Formula/rtl_433.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Rtl433 < Formula
+  desc "Tool to talk to 433Mhz devices using RTL devices"
+  homepage "https://github.com/merbanan/rtl_433"
+  url "https://github.com/merbanan/rtl_433.git", :revision => "922a76a81820c68e2e5c55a428e5d22135a8be9a"
```
Comment (DomT4) [inline Library/Formula/rtl_433.rb]: We'll need a stable, tagged upstream release here. You'll need to ask them for one, I suspect.

Comment (labeneator) [inline Library/Formula/rtl_433.rb]: Created an issue: https://github.com/merbanan/rtl_433/issues/324

Comment (labeneator) [inline Library/Formula/rtl_433.rb]: https://github.com/merbanan/rtl_433/issues/324 created 0.01

**Comment by mikemcquaid** [comment]
Please follow the steps in the checklist.

**Comment by UniqMartin** [comment]
Closing for a lack of response/information; happy to reopen once there's some activity here.

**Pull request closed without merging**